lib/comments.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Checks for a previous bot comment, if found returns the commen
+ */
+async function checkComments(issues,pull){
+    const comments = await issues.getComments(pull)
+    let result 
+//iterates over all the comments
+  for (comment of comments['data']){
+    if(comment.user.login=='commitlint[bot]'&&comment.user.type=='Bot'){ //looks for the first comment made by the bot
```

Comment:
I'm not sure about this but it has to better a better way to check for bot's comment other that it's login name.

This won't work for self-deployed instance since app names are unique.

Reply:
I don't think you can have anything more unique than login and type, I think something like this could give us a better idea about the self-deployed instances

https://developer.github.com/v3/apps/#get-the-authenticated-github-app
http://octokit.github.io/rest.js/#api-Apps-get
What do you think?

Reply:
@ahmed-taj What do you think about https://developer.github.com/v3/apps/#get-the-authenticated-github-app & http://octokit.github.io/rest.js/#api-Apps-get?